packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -4,15 +4,12 @@ pragma solidity 0.8.15;
 import { Blueprint } from "src/libraries/Blueprint.sol";
 import { Constants } from "src/libraries/Constants.sol";
 
-import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
-
 import { ISemver } from "src/universal/interfaces/ISemver.sol";
 import { IResourceMetering } from "src/L1/interfaces/IResourceMetering.sol";
 import { IBigStepper } from "src/dispute/interfaces/IBigStepper.sol";
 import { IDelayedWETH } from "src/dispute/interfaces/IDelayedWETH.sol";
 import { IAnchorStateRegistry } from "src/dispute/interfaces/IAnchorStateRegistry.sol";
 import { IDisputeGame } from "src/dispute/interfaces/IDisputeGame.sol";
-import { ISystemConfigV160 } from "src/L1/interfaces/ISystemConfigV160.sol";
```

Comment:
Why were we able to delete this?

Reply:
We can't it was just declared twice: https://github.com/ethereum-optimism/optimism/pull/12851/files#diff-32c676c7a2cc25dc934a436546eccc2c3484b44491cc1569dce382ffb13b84c7R28